### PR TITLE
[6962] end session automatically

### PR DESCRIPTION
**Pull request by Rineee**
Will also write tests, but wanted to check first if this is good?

### CODE REVIEW

**Review by fuzzylogic2000 — APPROVED** [review]
Yes, that works nicely!

**Comment by Rineee** [comment]
> Yes, that works nicely!

Cool, will add test then!

**Review by fuzzylogic2000 — APPROVED** [review]
Very nice! Only one question.

File: meinberlin/config/settings/base.py [hunk 1]
```diff
@@ -15,6 +15,8 @@
 
 from django.utils.translation import gettext_lazy as _
 
+import meinberlin.apps.votes.middleware
```
[the remark this answers is not in the document]
Comment (Rineee) [reply meinberlin/config/settings/base.py:18]: Good catch! It seems you forgot last time, at least I have not seen it .. will remove!

[the remark this answers is not in the document]
Comment (Rineee) [reply meinberlin/config/settings/base.py]: @fuzzylogic2000 done

**Pull request merged**